tools/pipelines/templates/build-npm-package.yml

```diff
@@ -264,16 +264,11 @@ stages:
                 fi
               fi
 
-              if [[ "$(release)" != "none" ]] && [[ "$(release)" != "" ]]; then
+              if [[ "$(release)" != "none" ]] || [[ "$(release)" != "" ]]; then
```

Comment:
Not sure this condition makes sense. It's always `true`, since `$(release)` can only be one thing so it will always be different from at least one of `"none"` and `""`. Did we miss changing the operators to equals?